### PR TITLE
Add fortran support

### LINKED ISSUE #70: add fortran support
add support for fortran package manager (fpm)

### PR DESCRIPTION
**Pull request by mustafasoylu**
add fortran support to somesy
resolves #70

### CODE REVIEW

**Comment by mustafasoylu** [comment]
@apirogov updated docs too, ready for review

File: README.md [hunk 1]
```diff
@@ -168,16 +168,18 @@ Here is an overview of all the currently supported files and formats.
...
 |                |        | | CITATION.cff                  | ✓      |
-|                |        | | codemeta.json                 | ✓(3.)  |
+|                |        | | codemeta.json                 | ✓(4.)  |
 
 **Notes:**
 
 1. note that `somesy` does not support setuptools *dynamic fields*
 2. `package.json` only supports one author, so `somesy` will pick the *first* listed author
-3. unlike other targets, `somesy` will *re-create* the `codemeta.json` (i.e. do not edit it by hand!)
+3. `fpm.toml` only supports one author and maintainer, so `somesy` will pick the *first* listed author and maintainer
```
Comment (apirogov) [inline README.md:181]: As currently the format is not specified (https://github.com/fortran-lang/fpm/issues/995), we technically could just concatenate author and maintainer name+emails, like e.g. `Jane Doe <mail1>, John Doe <mail 2>` ;) But maybe that's not a very useful idea... I just mean, technically, there is no restriction whatsoever for `author` and `maintainer` of fpm.toml, and some examples have something like `the XYZ project authors` as the `author` string.

File: src/somesy/cli/sync.py [hunk 2]
```diff
@@ -104,13 +104,13 @@ def sync(
...
-        "-M",
-        help="Do not sync Project.toml(Julia) file",
+        "-K",
+        help="Do not sync Project.toml(Julia) file (default: False)",
     ),
     julia_file: Path = typer.Option(
         None,
         "--julia-file",
-        "-m",
+        "-k",
```
Comment (apirogov) [inline src/somesy/cli/sync.py:113]: Let's use `L`/`l` for Julia - `j` is already taken but the file extension for Julia is `.jl` so `l` would make sense... whereas there is no `k` in `Julia` ;)

**Review by apirogov — CHANGES_REQUESTED** [review]
LGTM, except for julia CLI flag letter choice.

File: README.md (hunk 1 above)
Comment (mustafasoylu) [reply README.md:181]: I thought of that, too, but I decided to write only one person since the author and maintainer fields have singular key names.

**Review by apirogov — APPROVED** [review]

**Pull request merged**